### DISCUSSION TITLE
SQL Query Receiver

### DISCUSSION THREAD

**Discussion by mudim** [open-telemetry/opentelemetry-collector-contrib#31777] [General]
Hi all, hope all well. Looking for some clarification around sql query receiver, in context of logs.
Documentation is clear on how the state is maintained so that we don't fetch same records on restart of agent. However, when deployed as daemonset, want to understand how the state is replicated across receivers (one on each pod). Because, if the state is not replicated, wouldn't each of the receiver query same data, there by duplicating log records?

Cheers

**Comment by aishwaryaraimule21** [comment]
@dmitryax @crobert-1 I have a similar usecase. We are trying to use SQL Query Receiver for our Log Export pipeline. We store a few audit logs in Postgres and want to export it to a Syslog Server using OpenTelemetry.
The SQL Query Receiver for logs is in development stage. We have deployed it on a single collector instance as of now, can it be scaled out to multiple collector instances avoiding duplication?
